crates/sui-verifier-transactional-tests/tests/id_leak/leak_through_pack.mvir

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//# publish
```

Comment:
Could we add a few more tests for flagging on `pack` (e.g., leak via packing into into a field other than the first)?

Reply:
Other than the first would be a good one. General function passing stuff should be covered though